go.mod

```diff
@@ -18,11 +19,14 @@ require (
 	github.com/gobwas/glob v0.2.3
 	github.com/gofrs/flock v0.7.1
 	github.com/gosuri/uitable v0.0.4
+	github.com/jmoiron/sqlx v1.2.0
+	github.com/lib/pq v1.3.0
```

Comment:
@hickeyma I'm definitely not an expert in licenses but as far as I can see the only limitation here is:
```
The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
```
Which we'll be in respect with as the license is included in the dependency that will be pulled. Moreover this library is basically the core psql driver, I don't think there should be any constraints of use.

What do you think ?

Reply:
This is the [MIT license](https://opensource.org/licenses/MIT). My guess is that something about the first line is causing the automated parsing to not realize it. The [CNCF whitelist policy](https://github.com/cncf/foundation/blob/master/whitelist-policy.md) has the list of licenses and includes the MIT license. This should be ok to use from a license standpoint.

Reply:
@mattfarina How can you tell it is part of MIT license? I can only see this: https://github.com/lib/pq/blob/master/LICENSE.md

Reply:
@hickeyma That license is the MIT license. Compare it to https://opensource.org/licenses/MIT. There is something odd about the file so that GitHub doesn't auto-recognize it.

Reply:
I see what you mean now. Thanks for the clarification.